pkg/splunk/enterprise/standalone.go

```diff
@@ -158,7 +167,13 @@ func ApplyStandalone(client splcommon.ControllerClient, cr *enterprisev1.Standal
 		if err != nil {
 			return result, err
 		}
-		result.Requeue = false
+
+		// requeue the reconcile after polling interval if we had set the flag before
+		if cr.Status.AppContext.AppInfoStatus.NeedToCheckAfterPollInterval == true {
```

Comment:
Do we need this `NeedToCheckAfterPollInterval` in the `AppInfoStatus` or can this be a local variable?